assets/css/glimesh/components/stream.scss

```diff
@@ -5,6 +5,16 @@
     }
 }
 
+#video-column {
+    h5 {
+        position: relative;
+        top: 6px;
```

Comment:
I'm not sure I'm in love with bumping this up an arbitrary pixel amount. We should be able to remove the margin from the h5 and the vertical align it inside of it's container.

Reply:
it's not bumping it up, its bumping it down an additional 6px from the top, which is fine unless another element forces the container to resize. I think flexbox `align-items: center;` may be the only way to vertically center a container who's height can change, but I'm not sure how to do that without screwing up the whole

Reply:
ok so I think it needs to be
```
#video-column {
    .col-8{
	align-self:center;
    }
}

h5 {
	margin-bottom:0;
}
```
do we still want it to stay on one line and cut off the end with "..."?